picocolors.js

```diff
@@ -8,53 +8,51 @@ let isColorSupported =
 		(tty.isatty(1) && process.env.TERM !== "dumb") ||
 		"CI" in process.env)
 
-function formatter(open, close, replace = open) {
-	return (input) => {
+let formatter =
+	(open, close, replace = open) =>
+	(input) => {
```

Comment:
If we are going this way, should we change Prettier settings to reduce `(` and `)` for single argument function?

Reply:
That makes sense if we're all-in on squeezing every byte out of it: ed55367c0ffc701d4feeeb20c40bb7792b5390cd.